### PR TITLE
Issue/replace pay pal pro

### LINKED ISSUE #49: SI: Replace PayPal Express deprecated SOAP implementation
The current PayPal Express implementation has been depreciated and should be updated to the most current API PayPal has.

https://developer.paypal.com/docs/integration/direct/express-checkout/integration-jsv4/

### PR DESCRIPTION
**Pull request by VAlmea**
Here are present all the necessary changes to update the implementation of PayPal Pro. Also removed unnecessary classes and references. There is also a little modification to the class startPayPalExpressCheckout.cs that I realized was needed when I removed the reference to the old paypal_base.dll. 

close #49

### CODE REVIEW

File: Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs [hunk 1]
```diff
@@ -77,11 +76,115 @@ public HttpClient client()
...
+            expMonth = expMonth.Length == 1 ? $"0{expMonth}" : expMonth;
+            var orderConfirm = new OrderActionRequest()
+            {
+                PaymentSource = new PaymentSource()
+                {
+                    Card = new Card()
+                    {
+                        Number = creditCardNumber,
+                        Expiry = $"{expYear}-{expMonth}",
+                        Name = "John Doe",
```
Comment (WillStrohl) [inline Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs:172]: Why is this using "John Doe"?

**Review by WillStrohl — CHANGES_REQUESTED** [review]
This mostly looks good, but there is a single question so far about the code.  Also, I have another question that I'll send to you internally.

**Comment by WillStrohl** [comment]
@VAlmea  Oh, your close comment is attempting to close the previous PR and not the original Issue.  :)

File: Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs (hunk 1 above)
Comment (VAlmea) [reply Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs]: Solved

**Review by WillStrohl — APPROVED** [review]
Looks good, but needs to be tested still.

**Pull request merged**